generators/entity-client/files.js

```diff
@@ -244,7 +244,7 @@ function writeFiles() {
                     this.entityAngularName,
                     this.entityFolderName,
                     this.entityFileName,
-                    this.enableTranslation,
```

Comment:
Does removing this cause any issues?

Reply:
Was unused in the needle but I can double check it to be sure !

Reply:
Doesn't seem used, tested on i18n applications with entities so should be fine :)